maestro/plays/tasks.py

```diff
@@ -435,6 +435,8 @@ def get_registry_retry_spec(registry):
 
         When nothing is configured, no retries are attempted (by virtue of the
         'when' list being empty)."""
+        if registry is None:
+            registry = {}
```

Comment:
I think it would make more sense to return directly a "default retry spec object":

```python
_DEFAULT_RETRY_ATTEMPTS = 3
_DEFAULT_RETRY_SPEC = {'attempts': _DEFAULT_RETRY_ATTEMPTS, 'when': set([])}

# ...

    @staticmethod
    def get_registry_retry_spec(registry):
        if not registry:
            return _DEFAULT_RETRY_SPEC
        # ...
```

Reply:
Makes sense, corrected.